src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -347,12 +347,10 @@ public int getNumOfChoicesForMsq(String courseId, FeedbackParticipantType genera
             } catch (EntityDoesNotExistException e) {
                 Assumption.fail("Course disappeared");
             }
-        } else {
```

Comment:
Do it for the whole method - there should be no else ifs.

Reply:
I think I misunderstood what you meant. Do you mean the following?

```java
    public int getNumOfChoicesForMsq(String courseId, FeedbackParticipantType generateOptionsFor) {
        if (generateOptionsFor == FeedbackParticipantType.NONE) {
            return msqChoices.size();
        }

        if (generateOptionsFor == FeedbackParticipantType.STUDENTS
                || generateOptionsFor == FeedbackParticipantType.STUDENTS_EXCLUDING_SELF) {
            List<StudentAttributes> studentList = StudentsLogic.inst().getStudentsForCourse(courseId);
            int sizeOfStudentlist = studentList.size();

            return generateOptionsFor == FeedbackParticipantType.STUDENTS ? sizeOfStudentlist : sizeOfStudentlist - 1;
        }

        if (generateOptionsFor == FeedbackParticipantType.TEAMS
                || generateOptionsFor == FeedbackParticipantType.TEAMS_EXCLUDING_SELF) {
            try {
                List<TeamDetailsBundle> teamList = CoursesLogic.inst().getTeamsForCourse(courseId);
                int sizeOfTeamlist = teamList.size();

                return generateOptionsFor == FeedbackParticipantType.TEAMS ? sizeOfTeamlist : sizeOfTeamlist - 1;
            } catch (EntityDoesNotExistException e) {
                Assumption.fail("Course disappeared");
            }
        }
        List<InstructorAttributes> instructorList = InstructorsLogic.inst().getInstructorsForCourse(courseId);

        return instructorList.size();
    }
```

Reply:
Yup!

Reply:
I've push up the changes. What's the difference between this version and the previous version?

Reply:
👍 It just looks slightly neater I think